bin/kevm

```diff
@@ -124,18 +124,22 @@ run_krun() {
 run_kast() {
     local output_mode
 
-    output_mode="${1:-kore}" ; shift
+    if [ $# -eq 0 ]; then
+        output_mode=kore
+    else
+        output_mode="$1"; shift
+    fi
 
     case "${run_file}-${output_mode}" in
-        *.json-kast) execute kast-json.py "${run_file}" "${cSCHEDULE_kast}" "${cMODE_kast}" "${cCHAINID_kast}"
-                     ;;
         *.json-kore) if ! ${pyk}; then
                          execute kore-json.py "${run_file}" "${cSCHEDULE_kore}" "${cMODE_kore}" "${cCHAINID_kore}"
                      else
                          check_k_install
                          gst-to-kore "${run_file}" --schedule "${schedule}" --mode "${mode}" --chainid "${chainid}" "$@"
                      fi
                      ;;
+        *.json-kast) fatal "kast output is not supported for .json input"
+                     ;;
         *)           check_k_install ; kast --definition "${backend_dir}" "${run_file}" --output "${output_mode}" "$@"
                      ;;
```

Comment:
Do we need it to be unsupported? Or could we just fallback to `kast`?

```suggestion
        *.json-kast) check_k_install ; kast --definition "${backend_dir}" "${run_file}" --output "${output_mode}" --sort JSON "$@"
                     ;;
        *)           check_k_install ; kast --definition "${backend_dir}" "${run_file}" --output "${output_mode}" "$@"
                     ;;
```

Reply:
Also, now that I'm looking at it, gst-to-kore (and what `kore-json.py` and `kast-json.py` did before) are fundamentally different than this call to `kast`. This call to `kast` just parses a term, `gst-to-kore` builds an initial configuration. They probably should be pulled into their own functions.

Reply:
Going ot merge this because my point above doesn't really matter, no one uses this  functionality anyway.